### PR TITLE
Append timestamps to logs

### PR DESCRIPTION
**Pull request by axil**
This is the second advent of PR #333

### CODE REVIEW

File: supervisor/tests/test_options.py [hunk 1]
```diff
@@ -571,6 +571,7 @@ def test_options(self):
         self.assertEqual(proc1.startretries, 10)
         self.assertEqual(proc1.uid, 0)
         self.assertEqual(proc1.stdout_logfile, '/tmp/cat.log')
+        self.assertEqual(proc1.stdout_append_timestamp,False)
```
Comment (harpcio) [inline supervisor/tests/test_options.py:574]: ,___space___

Comment (axil) [reply supervisor/tests/test_options.py]: done

**Comment by mnaberez** [comment]
> This is the second advent of PR #333

Please don't open exact duplicates of existing PRs that have been closed.

**Comment by axil** [comment]
@mnaberez That PR was closed due to inactivity because noone was willing to fix obvious errors in it. Well, I did fix them.

This feature (https://github.com/Supervisor/supervisor/issues/553) is still open and eagerly sought for.

If you have any objections to the patch itself, please clarify them.

**Comment by mnaberez** [comment]
> If you have any objections to the patch itself, please clarify them.

https://github.com/Supervisor/supervisor/pull/333#issuecomment-50087335

**Comment by polmonso** [comment]
@axil will you have time to add the tests? I might give it a go if I find a moment

**Comment by mnaberez** [comment]
Closing due to inactivity.  Original request #553 will remain open.

**Comment by FazlanFarook** [comment]
+1

**Pull request closed without merging**